### ISSUE TITLE
Implement frame application design using iX

### ISSUE THREAD

**Issue by lamara11** [amosproj/amos2024ss01-xcelerator-demo-app#28]
## User story

1. As a developer
2. I want to implement the Frame Application Design using components from [ix.siemens.io](https://ix.siemens.io/). This means that generic layout components like [Frame Application](https://ix.siemens.io/docs/controls/application-frame/application), colors, button controls etc. are given by the framework, but developers are free to move in those boundaries.

Here is an example:

[Starting point](https://stackblitz.com/edit/iryzgw?embed=1&file=src%2Fapp%2Fapplication.html)
 see issue Q&A - 30. April 2024  #27 

## Acceptance criteria

- [ ] The Frame Application Design is implemented using iX components.
- [ ] The design adheres to the guidelines provided by the framework.
- [ ] Generic layout components such as Frame Application, colors, button controls, etc., are utilized from the framework.
- [ ] The implemented design is visually consistent with the provided sample.
- [ ] Design is discussed with and approved by the IP. 

## Definition of done (DoD)

- [ ] Added only after week 5
- [ ] The same for all features
- [ ] Here goes the project specific part

## DoD general criteria

- [ ] Feature has been fully implemented
- [ ] Feature has been merged into the mainline
- [ ] All acceptance criteria were met
- [ ] Product owner approved features
- [ ] All tests are passing
- [ ] Developers agreed to release

**Comment by sn0wcat** [comment]
if you pick up this task - this is the way to make the frame application use 100% of browser height:

```css
:host {
  display: block;
  width: 100%;
  height: 100vh;
}
```

see also  demo on stackblitz [https://stackblitz.com/edit/iryzgw?file=src%2Fapp%2Fapp.component.css](https://stackblitz.com/edit/iryzgw?file=src%2Fapp%2Fapp.component.css)

**Comment by sn0wcat** [comment]
The story for the application:

A water tank management company is managing water tanks installed in different locations. They can observe the water tanks and by problems issue work orders to repair or replace something. 

we are going to have these screens in the application:

- [ ] Issue List
- [ ] Facility Detailed View
- [ ] Create Work Order View
- [ ] List of all facilites view
- [ ] List of all open work orders view


![AMOS drawio](https://github.com/amosproj/amos2024ss01-xcelerator-demo-app/assets/1426617/ba45ca8c-dc67-4e8c-84cd-8a5d77fb45d9)
